openreview/api/client.py

```diff
@@ -1531,6 +1532,23 @@ def get_grouped_edges(self, invitation=None, head=None, tail=None, label=None, g
         json = response.json()
         return json['groupedEdges'] # a list of JSON objects holding information about an edge
 
+    def get_edges_archive(self, invitation):
```

Comment:
could we call this `get_archived_edges()`. I think it's a bit more descriptive

Reply:
I agree

Reply:
Done!